addons/l10n_es_edi_tbai/models/l10n_es_edi_tbai_certificate.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from base64 import b64decode
+from OpenSSL import crypto
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.serialization import pkcs12
+from odoo import models
+
+
+class Certificate(models.Model):
+    _inherit = 'l10n_es_edi.certificate'
+
+    # -------------------------------------------------------------------------
+    # HELPERS
+    # -------------------------------------------------------------------------
+
+    def _get_key_pair(self):
+        self.ensure_one()
+
+        if not self.password:
+            return None, None
+
+        private_key, certificate, dummy = pkcs12.load_key_and_certificates(
+            b64decode(self.with_context(bin_size=False).content),  # Without bin_size=False, size is returned instead of content
+            self.password.encode(),
+            backend=default_backend(),
+        )
+
+        return private_key, certificate
+
+    def _get_p12(self):
+        # Cryptography's pkcs12 does not contain issuer data ? TODO alternative without OpenSSL
+        return crypto.load_pkcs12(b64decode(self.with_context(bin_size=False).content), self.password.encode())
```

Comment:
Use _get_key_pair -> key, cert
then `cert.serial_number` for the serial number
and `cert.issuer.rdns` for the issuer

```suggestion        
```

Reply:
https://cryptography.io/en/latest/hazmat/primitives/asymmetric/serialization/#cryptography.hazmat.primitives.serialization.pkcs12.load_pkcs12
What a shame

Reply:
Thank you @JulienAlardot, I managed to remove the OpenSSL dependency on the EC branch and will update this PR accordingly.
For future reference, most helpful method (which I missed last time I investigated) was
https://cryptography.io/en/latest/x509/reference/#cryptography.x509.Name.get_attributes_for_oid